tests/microbenchmarks/bench_moe_comm.py

```
@@ -0,0 +1,738 @@
+
+
+def _time_dispatch_and_combine(
+    backend: Communication,
+    *,
+    hidden_states: torch.Tensor,
+    hidden_states_sf: Optional[torch.Tensor],
+    token_selected_slots: torch.Tensor,
+    token_final_scales: Optional[torch.Tensor],
+    all_rank_num_tokens: List[int],
+    hidden_size: int,
+    warmup: int,
```

Ruff (0.14.14)

Unused function argument: `warmup`

(ARG001)